index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- ABOT HEADER -->
+    <header class="nav-bar">
+        <a href="#" class="logo">My Logo</a>
+        <nav class="nav-section">
+            <div class="btn">
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+            </div>
+        </nav>
+    </header>
+    <!-- ABOUT MYSELF -->
+    <section class="about">
+        <h1 class="about-title">I’m Martin.Glad to see you!</h1>
```

Comment:
- [OPTIONAL] I would be perfect if you could break this line in order to make it look perfectly as we have it on the Figma design across different screen sizes.
```
   <h1 class="about-title">I’m Martin. <br> Glad to see you!</h1>
```